<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HELP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeDoor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CLOSE DOOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openDoor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN DOOR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PASSENGER VIEW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elevatorFire">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>FIRE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Floor Panel</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>-10</y>
      <width>20</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Elevator Panel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="elevatorLCD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="floorDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Control System</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="initialFloorPick">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="passengerOne">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="passengerTwo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="passengerThree">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overloadSystem">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Overload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="poweroutSystem">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fireSystem">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blockSystem">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Block</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSystem">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="elevatorDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set initial Floor</string>
    </property>
   </widget>
   <widget class="QWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>111</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="countElevators">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
